components/brave_ads/core/internal/ml/pipeline/text_processing/embedding_processing.cc

```diff
@@ -24,32 +24,29 @@
 namespace brave_ads::ml::pipeline {
 
 // static
-std::unique_ptr<EmbeddingProcessing> EmbeddingProcessing::CreateFromValue(
-    base::Value resource_value,
-    std::string* error_message) {
-  DCHECK(error_message);
```

Comment:
'#include "base/check.h"` can be removed

Reply:
Done.